src/map/lapack2flash/README

```diff
@@ -0,0 +1,57 @@
+
+lapack2flash
+============
+
+This directory includes lapack2flash layer which provides backward compatibility
+to FORTRAN LAPACK interface using FLASH. The entire LAPACK source files are converted to C
+files using code translator F2C pointed to in the lapack2flame directory.
+However, libf2c is not required when libflame is linked.
+
+Directory structure
+-------------------
+
+lapack2flash
+  - check               :: Symbolic link to the directory in lapack2flamec/
+  - f2c                 :: Symbolic link to the directory in lapack2flamec/
+
+  + FLASH_*.c           :: LAPACK interfaces that redirect functionality to libflame using FLASH calls.
+
+
+How to use
+----------
+
+configuration:
+
+  Modify the default run-conf/run-configure.sh with the following changes
+
+  ./configure \
+            --enable-lapack2flash \
+            --enable-supermatrix \
+            --enable-hip \
```

Comment:
let's remove that - HIP isn't and shouldn't be a hard requirement for this.

Reply:
These 4 comments are fixed in https://github.com/iotamudelta/libflame/pull/2/commits/3e980dee7b0eed7b8e87ce6d723863a7bd34f2e6